examples/README.md

```diff
@@ -9,6 +9,7 @@ Overview
 | <a href="Call out to an HTTP endpoint and cache the response.policy.xml">Call out to an HTTP endpoint and cache the response</a>                                                   | Enrich requests by calling HTTP endpoint before calling a backend service                                                                                                                                                             |
 | <a href="Filter response content based on product name.policy.xml">Filter response content based on product name</a>                                                               | Filter data elements from the response payload based on the product associated with the request                                                                                                                                       |
 | <a href="Forward gateway hostname to backend for generating correct urls in responses.policy.xml">Forward gateway hostname to backend for generating correct urls in responses</a> | Add a Forwarded header in the inbound request to allow the backend API to construct proper URLs                                                                                                                                       |
+| <a href="Forward Azure Event Grid Event.xml">Forward Azure Event Grid Event</a>                                                   | Forwards an Azure Event Grid event data payload to a downstream system and adds the event metadata to the HTTP headers.<br /><br />Good to know:<br /><ul><li>This policy also handles the initial security handshake with Azure Event Grid to prove ownership</li><li>This policy only supports one event per payload, if you want to support more you'll need to change it</li></ul>|
```

Comment:
Please keep the description short. Additional info can go to comments in the policy document.

Reply:
Sure, no problem - This is fixed now.